CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/io/ParameterIOUtilities.java

```diff
@@ -205,24 +208,21 @@ public static void loadParameters(final DataAccessPane dataAccessPane) {
 
                 // Groups all the parameters in to the plugin groups. Common parameters
                 // are based on the plugin name that is before the first '.' in the key values
-                final Map<String, Map<String, String>> ppmap = loadedParameter.toPerPluginParamMap();
```

Comment:
Is that `toPerPluginParamMap` used anymore? If not. Then remove it

Reply:
Ok. Thanks. I can remove that method and the unit test for that method as well.